app-frontend/src/app/pages/lab/run/run.controller.js

```diff
@@ -165,10 +176,12 @@ export default class LabRunController {
                         this.sideBySideControl.remove();
                         this.sideBySideAdded = false;
                     }
-                    this.$timeout(() => {
-                        m.map.invalidateSize();
-                        this.fitSceneList(this.inputs[0].id);
-                    });
+                    if (!this.alreadyFitted) {
```

Comment:
Can this be something more specific to the map? "Already fit" on the page that used to be models, which can also be fit, is just so much semantic overloading